### ISSUE TITLE
[REQUEST] Remember video position

### ISSUE THREAD

**Issue by SnowGabe** [YePpHa/YouTubeCenter#1547]
I wish that Youtubecenter would remember video position and continue playback automatically from where you left of.

**Comment by Piter432** [comment]
There's a nice extension for Firefox for desktop, which can be configurate
to user needs, but can't remember now it's name. Maybe someone who are
active here has this extension abd can tell more about it.

**Comment by rayman89** [comment]
If you want to see a really long video and save it for later just right click the video and select copy URL with time stamp then you can save that link on a text file or if you plan to close firefox and recover the session later just paste the link on the same tab you are viewing the video and close firefox.

An other alternative and this works for everything not just youtube is that you can hibernate your pc this actually shuts down the pc (the pc will be turned off) and when you turn it on it will start the pc just like you left it. With all the windows open all the videos where they were all the programs that were running will be running etc. 

If you have hibernation disabled you can enabled by pressing the windows key + R and pasting this "powercfg.exe /hibernate on" then hibernate will appear as an option when you hover over the > next to the shutdown Icon on the windows menu.

**Comment by ErdeFB** [comment]
Better not enable hibernate unless your PC's power supply actually supports it and if it does, Windows usually enables it by itself. ;)

**Comment by rayman89** [comment]
HIbernation is handled by the OS not the PSU. The only thing that you have to be sure is that you have enough disk space. You need to have as much GB free as RAM you have on you HDD or SDD to create the pagefile and that's all. It does not take more electricity either to "wake up" from hibernation either.

**Comment by ErdeFB** [comment]
Hibernation is not indeed handled by the PSU, but if your PSU does not support hibernate, it will never wake up since it does not actually keep your computer powered in any way.

**Comment by rayman89** [comment]
You are a bit confused. When you hibernate your pc you can unplug it. It does not consume any power at all cause the pc is shuted down. When you turn on your computer for all the hardware it will be like you just normally turned on the pc the only difference is that windows will notice that there is a file will all the dumped ram and open that file restoring the session.

**Comment by ErdeFB** [comment]
So it seems. I guess I am confusing it to sleep mode or something. Apologies.

**Comment by CyberMew** [comment]
This seems to be implemented by YouTube already if you are logged in (at least for me)

**Comment by Fofer** [comment]
Yes, CyberMew, I have noticed this too.  YouTube appears to be bookmarking our "last viewed" spots in watched videos' timelines, and when we revisit they continue where we left off...

**Comment by SnowGabe** [comment]
I don't think YouTube has 100% implemented this. And I can confirm "continue playback" doesn't work when video autoplay is stopped.

**Comment by Zarggg** [comment]
The last time I looked into that behavior, it seemed that YouTube was saving positions only in videos longer than 30 minutes. I haven't had opportunity to test whether this is still the case recently.
